quiche/src/recovery/mod.rs

```diff
@@ -270,6 +277,27 @@ impl Recovery {
         }
     }
 
+    #[cfg(feature = "qlog")]
+    pub fn maybe_qlog(
+        &mut self, qlog: &mut qlog::streamer::QlogStreamer, now: Instant,
+    ) {
+        if let Some(ev_data) = self.get_updated_qlog_event_data() {
+            qlog.add_event_data_with_instant(ev_data, now).ok();
+        }
+
+        if let Some(cc_state) = self.get_updated_qlog_cc_state(now) {
+            let ev_data = EventData::CongestionStateUpdated(
+                qlog::events::quic::CongestionStateUpdated {
+                    old: None,
```

Comment:
Given we already have the old state, is it worth attaching here for completeness? `self.get_updated_qlog_cc_state` could have return `(curr, prev)` states for use here, for example. Feel free to resolve if it wouldn't give much benefit though

Reply:
I'ld need an API to return both old and new.  Visualizations just based on the new state work pretty well.  Is old really needed?